README.md

```diff
@@ -26,6 +44,7 @@ timesToTest: 5, // optional, number of times to load the image default is 5
 threshold: 200, // optional, threshold in ms after which internet speed is considered slow
 image:  "http://www.google.com/images/phd/px.gif", //  optional, url of the tiny image to load
 allowEarlyExit: true // optional, if the first request takes greater than threshold*3 ms then the function exits with false
+verbose: false  // optional, if set, it returns an object with all calculated latency data. Overrides allowEarlyExit option (See Example #2 for usage)
```

Comment:
@Aendir Just need a comma here after "true". I'm not quite sure if and how I can make such tiny edits before merging.

Reply:
Done!